### DISCUSSION TITLE
How to create a public page accessible on public or unauthenticated ?

### DISCUSSION THREAD

**Discussion by nlvrendell** [filamentphp/filament#8843] [Help] [answered]
I am curious how to create a page where unauthenticated can access, like the login page? I've digged into the code of login and tried to create a new page using this command `php artisan make:filament-page Page` while also extends the SimplePage class as the same on Login while removing the Page class. 

Still no luck. Anyone who already tried this? Can you give me some adivice. Thanks!

**Comment by pboivin** [comment, marked as answer]
If your page extends `SimplePage`, you can use the following wrapper component in your Blade view:

```blade
<x-filament-panels::page.simple>
    ...
</x-filament-panels::page.simple>
```

Then, you can register the page in your `routes/web.php` like other pages of your app:

```php
Route::get('/my-page', MyPage::class)->name('my-page');
```

**Reply by nlvrendell** [reply]
Worked! Thanks

**Reply by almontasser** [reply]
But how to customize the theme for that page?

**Reply by vierilusen** [reply]
@almontasser 
Hi, just in case anyone else runs into the same issue — this worked for me.
You can add the following code inside the `mount()` method of your `Page class`:

```
$panel = Filament::getCurrentPanel();
$panel->boot();
```

**Comment by kofiarkoh** [comment]
Is there a way to achieve the same thing for a page which extends CreateRecord ?

**Reply by nlvrendell** [reply]
Hi @kofiarkoh, you cant extend CreateRecord on a public page, cause it requires auth. But I think you can create your own  custom form inside a public page. Look at the youtube link below how  to implement it.


https://www.youtube.com/watch?v=tsOeQQZ9YdY

**Reply by kofiarkoh** [reply]
thanks @nlvrendell , i got it!

**Reply by 404us3r** [reply]
hi guys! I'm new to this but can you help me. I already have a landing page and I want to use the default modal of breeze to call the filament login page. Is that possible?
